vite.browser.config.ts

```diff
@@ -0,0 +1,31 @@
+import { nodePolyfills } from 'vite-plugin-node-polyfills';
+import { mergeConfig } from 'vitest/config';
+import type { UserConfig } from 'vitest/config';
+
+import baseConfig from './vite.base.config';
+
+const config: UserConfig = {
+  plugins: [
+    nodePolyfills({
+      include: ['process', 'buffer'],
+      globals: {
+        Buffer: false,
+        global: true,
+        process: true,
+      },
+      protocolImports: false,
+    }),
+  ],
```

Comment:
Are these polyfills for the browser testing environment?

Reply:
They are indeed, browsers aren't shipped with these by default

Reply:
Same comment about moving browser-related stuff to its PR.